### ISSUE TITLE
Timeline editing features are not accessible via screen reader

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#597]

**Comment by thenickcox** [comment]
Can we have more info here?

**Comment by ragesoss** [comment]
In general, anything clickable should either be an <a /> or a <button />. Those are the things I know trip up a screen reader. There are probably others as well, though.

**Comment by thenickcox** [comment]
Hmmm…well, the first part definitely makes sense. If that's the case, shall we change the title of this issue? The idea of there "probably [being] others" makes it difficult to know when this issue has been addressed.

**Comment by thenickcox** [comment]
Looks like we can [add ARIA roles](https://www.marcozehe.de/2013/04/24/easy-aria-tip-6-making-clickables-accessible/) to spans for a11y?

**Comment by ragesoss** [comment]
If we fix those known first, then I'll ask Helaine (the Wiki Ed staffer who uses a screen reader) to try it again, and I can work with her to pin down what is and isn't accessible.

**Comment by ragesoss** [comment]
Fixed with bc01e7c05be332114a22ca8ccf6d06e03060a706

`role` and `aria-label` attributes were the relevant things.
